### ISSUE TITLE
Build and Release uYouPlus Workflow Error "Failed to download uYou"

### ISSUE THREAD

**Issue by Chimbul** [qnblackcat/uYouPlus#1031]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

❌ No, I don't think this is a bug. I will explain below

### My question

I was able to build uYouPlus from my fork until last week. I forked this git just to be able to build the latest uYouPlus. I have made no code changes. I have tried multiple time. This is the error message.

![Screenshot 2023-03-20 134803](https://user-images.githubusercontent.com/8114661/226424495-2dce7189-102d-421a-bd91-e8725f3e6257.png)

Thanks

**Comment by qnblackcat** [comment]
please try again

**Comment by Chimbul** [comment]
I tried again and the build past the error and downloaded uYou. Some steps later in the build I had another error:

![Screenshot 2023-03-26 195910](https://user-images.githubusercontent.com/8114661/227813780-3d492ae5-009e-40f7-9121-e8aa0b86360a.png)

I did some research and tracked down the issue and found it is something related to Return-YouTube-Dislikes. The fix is to update Return-YouTube-Dislikes submodule to the latest version/commits. Actually, the latest commit is the [fix](https://github.com/PoomSmart/Return-YouTube-Dislikes/commit/4145aa71bccf550205f1ec5011252379b637da4a). After that, everything worked as expected.Thanks
